File: libs/perllib/LoxBerry/testlog.pl
#!/bin/perl

use LoxBerry::Log;

print "Hallo\n";

# my $log = LoxBerry::Log->new ( logdir => "$lbslogdir", name => 'test', package => 'Test', loglevel => 3);
my $log = LoxBerry::Log->new ( filename => "$lbslogdir/test.log", name => 'test', package => 'Test', loglevel => 3, append => 1);


LOGSTART "My test logging";

$log->ERR("Das ist ein Fehler!");
LOGERR "Das ist auch ein Fehler!";


LOGDEB "Debug";
LOGINF "Info";
LOGOK "OK";
LOGWARN "Warning";
LOGERR "Error";
LOGCRIT "Critical";
LOGALERT "Alert";
LOGEMERGE "Emergency";

LOGEND "Process finished";




# print "Filename is " . $log->filename . "\n";

File: libs/phplib/testing/testlog.php
#!/usr/bin/env php
<?php
require_once "loxberry_log.php";

$mylog = LBLog::newLog(array("addtime" => "1", "name" => "PHPLog", "package" => "Test", "logdir" => $lbslogdir));
LOGSTART("Das ist mein gl PHP Log");
LOGDEB("Das ist eine gl Debug Message");
LOGINF("Das ist eine gl Info Message");
LOGOK("Das ist eine gl OK Message");
LOGWARN("Das ist eine gl Warning Message");
LOGERR("Das ist eine gl Error Message");
LOGCRIT("Das ist eine gl Critical Message");
LOGALERT("Das ist eine gl Alert Message");
LOGEMERG("Das ist eine gl Emergency Message");
LOGTITLE("New title");
LOGEND("Das ist das gl Log Ende");

?>
